src/utils/shared/k8s/apply.go

```
@@ -30,6 +30,7 @@ import (
 	"strings"
 
 	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/meta"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)